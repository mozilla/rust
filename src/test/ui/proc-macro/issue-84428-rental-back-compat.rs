// check-pass
// aux-build:test-macros.rs

include!("pretty-print-compat-hack/rental-0.5.5.rs");
