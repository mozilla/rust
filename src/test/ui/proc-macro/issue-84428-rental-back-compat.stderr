warning: using outdated version of `rental` crate
  --> $DIR/pretty-print-compat-hack/rental-0.5.5.rs:7:6
   |
LL | enum ProceduralMasqueradeDummyType {
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(proc_macro_back_compat)]` on by default
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #83125 <https://github.com/rust-lang/rust/issues/83125>
   = note: Versions of the `rental` crate below `0.5.6` will eventually stop compiling. Please update to the latest version of `rental`

warning: 1 warning emitted

Future incompatibility report: Future breakage date: None, diagnostic:
warning: using outdated version of `rental` crate
  --> $DIR/pretty-print-compat-hack/rental-0.5.5.rs:7:6
   |
LL | enum ProceduralMasqueradeDummyType {
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(proc_macro_back_compat)]` on by default
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
   = note: for more information, see issue #83125 <https://github.com/rust-lang/rust/issues/83125>
   = note: Versions of the `rental` crate below `0.5.6` will eventually stop compiling. Please update to the latest version of `rental`

